### PR TITLE
Add normalizer detail page to katalogus

### LINKED ISSUE #1816: Improve "scan history tables" on boefje and normalizer detail pages
Currently we use one table for both kinds of plugins, while the content could (and should) vary between the two types. We already revised this on the task list and it would be nice if we could have the same columns on the plugin detail pages.

### PR DESCRIPTION
**Pull request by TwistMeister**
### Changes
Refactored the plugin detail view to two separate views. One dedicated boefje detail view and a normalizer detail view. They both inherit the PluginDetailView.

### Issue link
Closes: #1288 and closes #1816

### Proof
image
---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [ ] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* The new boefje and normalizer detail pages load and are generally in an OK shape

### What doesn't work:
* The normalizer detail page doesn't show the purple breadcrumb bar
image
* Attempting to delete all Boefje settings from the detail page results in the below error:
image

File: rocky/katalogus/views/plugin_settings_delete.py [hunk 1]
```diff
@@ -55,7 +54,7 @@ def get_context_data(self, **kwargs):
 
     def get_success_url(self):
         return reverse(
-            "plugin_detail",
+            "boefje_detail",
             kwargs={
                 "organization_code": self.organization.code,
                 "plugin_type": self.plugin.type,
```
Comment (Donnype) [inline rocky/katalogus/views/plugin_settings_delete.py:60]: Regarding the issue from @Darwinkel, my guess is that you still need to remove this line

File: rocky/rocky/templates/partials/task_history.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                                {% elif task.status.value == "completed" %}
+                                    <i class="icon positive"></i>&nbsp;{{ task.status.value|capfirst }}
+                                {% else %}
+                                    <i class="icon neutral"></i>&nbsp;{{ task.status.value|capfirst }}
+                                {% endif %}
+                            </td>
+                            <td>{{ task.created_at }}</td>
+                            <td>
+                                {% if task.type == "boefje" %}
+                                    <a href="{% ooi_url "ooi_detail" task.p_item.data.input_ooi organization.code %}">{{ task.p_item.data.input_ooi }}</a>
```
Comment (underdarknl) [inline rocky/rocky/templates/partials/task_history.html:101]: include the create_time in the url here.

File: rocky/rocky/templates/partials/task_history.html [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                                {% elif task.type == "normalizer" %}
+                                    <a href="{% ooi_url "ooi_detail" task.p_item.data.raw_data.boefje_meta.input_ooi organization.code %}">{{ task.p_item.data.raw_data.boefje_meta.input_ooi }}</a>
```
Comment (underdarknl) [inline rocky/rocky/templates/partials/task_history.html:103]: Same here

File: rocky/rocky/templates/partials/task_history.html [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                                    <i class="icon negative"></i>&nbsp;{{ task.status.value|capfirst }}
+                                {% elif task.status.value == "completed" %}
+                                    <i class="icon positive"></i>&nbsp;{{ task.status.value|capfirst }}
+                                {% else %}
+                                    <i class="icon neutral"></i>&nbsp;{{ task.status.value|capfirst }}
+                                {% endif %}
+                            </td>
+                            <td>{{ task.created_at }}</td>
+                            <td>
+                                <a href="{% url "boefje_detail" organization.code task.p_item.data.raw_data.boefje_meta.boefje.id %}">{{ task.p_item.data.raw_data.boefje_meta.boefje.id }}</a>
```
Comment (underdarknl) [inline rocky/rocky/templates/partials/task_history.html:154]: Same here. We need to link to the object at a time when we are sure it existed.

File: rocky/katalogus/views/plugin_settings_delete.py (hunk 1 above)
Comment (TwistMeister) [reply rocky/katalogus/views/plugin_settings_delete.py]: Yes, that indeed was the case.

**Review by ammar92 — DISMISSED** [review]
No remarks

**Review by underdarknl — APPROVED** [review]

**Pull request merged**